charts/ratify/templates/deployment.yaml

```diff
@@ -51,13 +51,18 @@ spec:
           ports:
             - containerPort: 6001
           volumeMounts:
-            - mountPath: "/usr/local/ratify-certs"
-              {{- if .Values.akvCertConfig.enabled }}
+            - mountPath: "/usr/local/ratify-certs/notary"
+            {{- if .Values.akvCertConfig.enabled }}
```

Comment:
How would this work if the user stored the cosign cert in keyvault? Would it still mount in the `/notary` directory? (don't know if this a valid scenario we even need to consider but though I'd point it out)

Reply:
I have the same concern. And the next notation release would need a few different directories to store certs. Do we want users to configure multiple AKVs?

Reply:
this is a good point. we ll have to investigate if our AKV scenario support cosign today. ( i m not sure if this was ever validated). In my previous testing, csi driver mounts a .Crt cert (good with notary) , but the Cosign verifier takes a PEM file.
My question is, how to configure secret store to export PEM .pub file,  or are keys stored differently in azure key vault that will automatically e different format when loaded by csi driver?

Reply:
Logged issue https://github.com/deislabs/ratify/issues/402, so we could investigate later, for now , mounting to notary cert location